README.md

```diff
@@ -297,60 +281,209 @@ The complete list of transformations supported and their usage in ImageKit can b
 | effectContrast | effectContrast(flag: Bool) | e-contrast |
 | effectGray | effectGray(flag: Bool) | e-grayscale |
 | original | original() | orig |
+| Raw param string | raw(params: String) | - |
 
 </details>
 
-### File Upload
-The SDK provides a simple interface using the `ImageKit.shared.uploader().upload(...)` method to upload files to the ImageKit Media Library. It accepts all the parameters supported by the [ImageKit Upload API](https://docs.imagekit.io/api-reference/upload-file-api/client-side-file-upload#request-structure-multipart-form-data).
+### Responsive image loading
+ImageKit URL constructor can be set to input any Android View and determine the parameters for image height and width based on the dimensions of the view. If the view's dimensions are not set at the time, it uses the display's dimensions as the fallback. It also sets the DPR of the image automatically to match that of the device display.
```

Comment:
This document is for ios, it has instances of android being written here

Reply:
Fixed